LICENSE

```diff
@@ -0,0 +1,176 @@
+                                 Apache License
```

Comment:
@ptrelford what should the license be ?

Reply:
I took it from here: http://tickspec.codeplex.com/license

Reply:
Hm, how did I drop it, apologies!

Reply:
I have just noticed it when converting the nuspec to paket.template. I also fixed the link to the project there.